src/datasources/transaction-api/transaction-api.service.ts

```diff
@@ -309,6 +310,20 @@ export class TransactionApi implements ITransactionApi {
     }
   }
 
+  async postConfirmation(
+    safeTxHash: string,
+    addConfirmationDto: AddConfirmationDto,
+  ): Promise<unknown> {
+    try {
+      const url = `${this.baseUrl}/api/v1/multisig-transactions/${safeTxHash}/confirmation`;
```

Comment:
Only caught this now but the endpoint should be `/confirmations` and not `/confirmation`

Reply:
Good catch! Fixed in [96df62d](https://github.com/safe-global/safe-client-gateway-nest/pull/293/commits/96df62d7aeebc277f37a44b7de47173de85628a1)